### PR TITLE
Trained Cellpose3 models

### PR DESCRIPTION
**Pull request by annayang26**
The code was edited based off of the `micromanager-gui-calcium` branch

1. Pushed three Cellpose3 models. The `cp_img8_epoch7000_py` has the best performance among the three models that I pushed in most cases. The other two models had a decent performance compared to the models (not published) trained with the same number of images but different training iterations. 
- cp_epoch6000_py: Cellpose3 model further trained with 6 images with 6000 training epochs
- cp_img8_epoch7000_py: Cellpose3 model further trained with 8 images with 7000 training epochs
- cp_img10_epoch6000_py: Cellpose3 model further trained with 10 images with 6000 training epochs

2. Uploaded the code that I used to train and load the models in case other users might want to train their own.

3. Uploaded graphs plotting the train and test losses for model performance evaluation

4. When loading the three custom cellpose models in `segmentation.py`, I specified the use of `torch.device('cpu')` (`line 304-309`).
** The custom models were trained with Python code to evaluate the model performance on a testing dataset. It was trained with CPU due to the warning on the Cellpose documentation. 
** For the models trained with the Cellpose GUI, this might not be an issue. Therefore, using `torch.device('cpu')` might not be necessary (**not yet reflected in the code**).
Screenshot 2025-03-03 at 11 55 00 AM

5. Reduced the number of output on `line 383` in the `segmentation.py` when calling `model.eval` from four to three. (`masks, _, _ = model.eval(cyto_frame, diameter=diameter, channels=channel)`)

---

CYTO3 MODEL:
Screenshot 2025-03-06 at 7 52 41 PM

CUSTOM MODEL:
Screenshot 2025-03-06 at 7 51 25 PM

### CODE REVIEW

**Comment by fdrgsp** [comment]
thanks @annayang26!!! The trained model does indeed work better🚀
